ftw/colorbox/browser/resources/colorbox.scss

```diff
@@ -0,0 +1,62 @@
+$cbox-button-background-color: #fff;
```

Comment:
I'd make those variables configurable by appending ` !default`, making it possible to define the variable before the default is defined here.

I'd also register it with
```scss
@include declare-variables(
    cbox-button-background-color,
    cbox-button-size);
```